include/grpc++/server_builder.h

```diff
@@ -127,7 +127,7 @@ class ServerBuilder {
       bool is_frequently_polled = true);
 
   //////////////////////////////////////////////////////////////////////////////
-  // RegisterService variants
+  // Less commonly used RegisterService variants
```

Comment:
`RegisterService()` is under this banner, and it's used very commonly.  It's only `RegisterAsyncGenericService()` that's less common.